packages/edit-site/src/components/global-styles/typography-panel.js

```diff
@@ -110,6 +109,26 @@ function useStyleWithReset( path, blockName ) {
 	return [ style, setStyle, hasStyle, resetStyle ];
 }
 
+function useFontSizeStyleWithReset( path, blockName ) {
```

Comment:
nit: `useFontSizeWithReset` or `useFontSize` would be more consistent with the other hooks in this file.

Reply:
Good point. On it.